timeline.md

```diff
@@ -9,26 +9,58 @@ css_id: timeline
 browser, Justin Samuel and Justin Cappos collaborate to design and publish an
 academic research paper on The Update Framework (TUF).
 
+**2011**: TUF Project moves to New York University Polytechnic
+(later NYU Tandon School of Engineering) when Justin Cappos accepts a post
+as an assistant professor in the Brooklyn, NY, school.
+
 **2013**: Justin Cappos, Trishank Kuppusamy, and Vladimir Diaz begin research
 into adapting and improving TUF for Python, Ruby, and other environments used
 for cloud computing.
 
+**2013**: The first of two Python Enhancement
+Proposals (PEP 458) dealing with TUF is published. "Surviving a Compromise of
+PyPI" details the integration of TUF into PyPI, the Python
+package manager.
+
 **2014**: Flynn.io independently implements TUF in the Go programming language.
 
-**2015**: Docker launches Notary. It also launches Docker Content Trust, which
-uses Notary to sign and verify container images.
+**2014**: PEP 480, a maximum security version of PEP 458, is published
+
+**2015**: Docker launches Notary, an implementation of TUF
+used to publish and manage trusted collections of content. It also launches
+Docker Content Trust, which uses Notary to sign and verify container images.
 
 **2016**: *Diplomat: Using Delegations to Protect Community Repositories*
-accepted to NSDI 2016.
+is accepted to NSDI 2016. Diplomat is the first of several TUF adaptations
+developed to address a specific identified problem in practice, in this case
+the need for faster registration of new projects on community repositories
+without sacrificing security. It also introduces the concept of delegation
+of trust.
 
 **2016**: A consortium including NYU Tandon (Cappos, Kuppusamy, Diaz, Awwad),
 theUniversity of Michigan Transport Research Institute (UMTRI), and the
-Southwest Research Institute (SWRI), begin developing Uptane, an evolution of
-TUF, to protect updates for vehicles from being easily compromised by rogue
+Southwest Research Institute (SWRI), begin developing Uptane, another evolution of
+TUF designed to protect updates for vehicles from being easily compromised by rogue
 nation-state attackers.
 
+**2016**: *Uptane: Securing Software Updates for Automobiles* is presented at
+escar 16.
+
+**2017**: Uptane is officially introduced at press events in Ann Arbor, MI, and
+Brooklyn, NY.
+
 **2017**: *Mercury: Bandwidth-Effective Prevention of Rollback Attacks Against
-Community Repositories* accepted to USENIX ATC 2017.
+Community Repositories* accepted to USENIX ATC 2017. Mercury is another TUF
+adaptation developed to protect rollback attacks on community repositories at
+a reasonable bandwidth cost.
+
+**2017**: Uptane is named one of the year's most important innovations in
+security by *Popular Science*
```

Comment:
This entry also doesn't end with a period like the others.

Reply:
Just a general comment---I think we need to distinguish between the "timeline" and the "news" section. I'm thinking "timeline" can contain the major steps forward for TUF, while the "news" listings should be more current interest, but not necessarily important milestones. Towards that end, I'm thinking that the "news" section should not go back very far. FYI, do we still want to have events from 2016 on that list? If they are important enough to still be up, shouldn't they be "Timeline" events?